hivemq-edge/src/frontend/src/api/hooks/useProtocolAdapters/useCreateProtocolAdapter.tsx

```diff
@@ -17,7 +17,8 @@ export const useCreateProtocolAdapter = () => {
     return appClient.protocolAdapters.addAdapter(adapterType, requestBody)
   }
 
-  return useMutation<unknown, ApiError, CreateProtocolAdapterProps>(createProtocolAdapter, {
+  return useMutation<unknown, ApiError, CreateProtocolAdapterProps>({
+    mutationFn: createProtocolAdapter,
     onMutate: (a) => {
```

Comment:
question(non-blocking): I don't know what `a` if you can it would be nice to put in the full verb. I guess it's `request` but unsure.